fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -931,6 +935,34 @@ public Frontend unprotectCheckFeExist(String host, int port) {
         return null;
     }
 
+    protected boolean checkFeExistByIpOrFqdn(String ipOrFqdn) throws UnknownHostException {
+        Pair<String, String> targetIpAndFqdn = NetUtils.getIpAndFqdnByHost(ipOrFqdn);
+
+        for (Frontend fe : frontends.values()) {
+            Pair<String, String> curIpAndFqdn;
+            try {
+                curIpAndFqdn = NetUtils.getIpAndFqdnByHost(fe.getHost());
+            } catch (UnknownHostException e) {
+                LOG.warn("failed to get right ip by fqdn {}", fe.getHost(), e);
+                if (targetIpAndFqdn.second.equals(fe.getHost())) {
```

Comment:
why don't check `!Strings.isNullOrEmpty(targetIpAndFqdn.second)` here?

Reply:
If the code reaches here, it means the targetIpAndFqdn.second cannot be null, but for code robust, add this check will be better. 
done